CRM/Admin/Form/Preferences/Mailing.php

```diff
@@ -63,4 +64,16 @@ public function postProcess() {
     parent::postProcess();
   }
 
+  /**
+   * @return array
+   */
+  public static function getAvailableFromEmailAddresses() {
+    $fromAddress[NULL] = ts('');
```

Comment:
I started to comment about how this is an incorrect use of `ts()` since empty strings can't be translated, but then I realized this entire function does not need to exist.

Reply:
I remember I added this function to include an empty option, `$fromAddress[NULL] = '';` so users can clear the select box.